cinema/models.py

```diff
@@ -35,12 +39,18 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_path(instance: "Movie", filename: str) -> pathlib.Path:
+    filename = f"{slugify(instance.title)}-{uuid.uuid4()}" + pathlib.Path(filename).suffix
+    return pathlib.Path("upload/movies") / pathlib.Path(filename)
```

Comment:
The `movie_image_path` function uses `pathlib.Path` to construct the file path. Django's `ImageField` expects a string path, not a `Path` object. Consider using `os.path.join` or converting the `Path` object to a string with `str()`.